addons/pos_self_order/models/pos_config.py

```diff
@@ -424,6 +424,8 @@ def action_open_wizard(self):
         self.ensure_one()
 
         if not self.current_session_id:
+            pos_session = self.env['pos.session'].create({'user_id': self.env.uid, 'config_id': self.id})
+            pos_session._ensure_access_token()
```

Comment:
i don't think you need this 2nd line. Can you please check if it works without it?